tests/commands/base/test_paths.py

```diff
@@ -9,6 +10,39 @@
 from .conftest import DummyCommand
 
 
+def test_path_is_realpath(tmp_path):
+    """Briefcase's data path matches its realpath.
+
+    For the Windows Store Python, filesystem interaction with
+    `%LOCALAPPDATA%` can be redirected to a sandboxed location. However,
+    `os.path.realpath()` will reveal such a redirection.
+    """
+    # %LOCALAPPDATA% seemingly isn't available in GH CI...
+    if user_dir := os.environ.get("USERPROFILE"):
+        tmp_path = Path(user_dir) / "AppData/Local"
+    data_path = tmp_path / "realpathbase"
+    command = DummyCommand(data_path=data_path)
+
+    assert command.data_path == Path(os.path.realpath(data_path))
```

Comment:
This test seems a bit broad/weak; if you're not on Windows, I t isn't needed, and if you *are* on Windows, it's dependent on the existence of the `USERPROFILE` variable.

Could we:
1. lock this down to a Windows-only test;
2. Add an explicit failure on Windows if the expected environment variable isn't present
3. Use try/finally to do cleanup, so we don't leave dirty state in the user's home folder

Also - is LOCALAPPDATA a complete substitute for USERPROFILE?

Reply:
Addressed. The test only runs on Windows now, will throw `KeyError` is `LOCALAPPDATA` isn't available, and deletes the directory.